lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -30,6 +30,7 @@ const _flatten = arr => [].concat(...arr);
 
 let lighthouseIsRunning = false;
 let latestStatusLog = [];
+let latestArtifacts = undefined;
```

Comment:
whats the lifecycle for this like for devtools and the extension? Is memory eventually cleared? The artifacts can be really big